std_msgs/Header header
ArmAction[] motions
---
bool success
